### PR TITLE
feat: [CHAOS-391]: Updating resources and terminologies

### PR DESCRIPTION
**Pull request by S-ayanide**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11686)

File: src/modules/20-rbac/pages/UserDetails/__tests__/mock.ts [hunk 1]
```diff
@@ -403,7 +403,7 @@ export const roleBindingsList = {
               'gitops_cluster_edit',
               'ccm_overview_view',
               'core_usergroup_view',
-              'chaos_chaosgitops_edit',
+              'chaos_chaosGameday_edit',
```
Comment (arkajyotiMukherjee) [inline src/modules/20-rbac/pages/UserDetails/__tests__/mock.ts:406]:
```suggestion
              'chaos_chaosgameday_edit',
```

File: src/modules/75-chaos/strings/strings.en.yaml [hunk 2]
```diff
@@ -3,9 +3,9 @@ homepage:
   slogan: Take your software build process to the next level using our Harness Chaos Engineering.
   learnMore: Learn more about Chaos Engineering
 chaosHub: ChaosHub
-chaosScenario: Chaos Scenario
+ChaosExperiment: Chaos Experiment
```
Comment (arkajyotiMukherjee) [inline src/modules/75-chaos/strings/strings.en.yaml:6]:
```suggestion
chaosExperiment: Chaos Experiment
```

File: src/modules/75-chaos/RouteDestinations.tsx [hunk 3]
```diff
@@ -107,12 +107,12 @@ AuditTrailFactory.registerResourceHandler(ResourceType.CHAOS_HUB, {
...
 
-AuditTrailFactory.registerResourceHandler(ResourceType.CHAOS_SCENARIO, {
+AuditTrailFactory.registerResourceHandler(ResourceType.CHAOS_EXPERIMENT, {
   moduleIcon: {
     name: 'chaos-main'
   },
-  moduleLabel: 'chaos.chaosScenario',
-  resourceLabel: 'chaos.chaosScenario',
+  moduleLabel: 'chaos.ChaosExperiment',
+  resourceLabel: 'chaos.ChaosExperiment',
```
Comment (arkajyotiMukherjee) [inline src/modules/75-chaos/RouteDestinations.tsx:115]:
```suggestion
  moduleLabel: 'chaos.chaosExperiment',
  resourceLabel: 'chaos.chaosExperiment',
```

File: src/modules/20-rbac/pages/UserDetails/__tests__/mock.ts [hunk 4]
```diff
@@ -916,14 +916,14 @@ export const roleBindingsList = {
...
-              'chaos_chaosgitops_view',
+              'chaos_chaosgameday_view',
               'core_setting_view',
               'core_governancePolicySets_edit',
               'core_environment_view',
               'core_environmentgroup_access',
               'gitops_cluster_edit',
               'core_usergroup_view',
-              'chaos_chaosgitops_edit',
+              'chaos_chaosGameday_edit',
```
Comment (arkajyotiMukherjee) [inline src/modules/20-rbac/pages/UserDetails/__tests__/mock.ts:926]:
```suggestion
              'chaos_chaosgameday_edit',
```

File: src/modules/20-rbac/pages/UserDetails/__tests__/mock.ts [hunk 5]
```diff
@@ -664,14 +664,14 @@ export const roleBindingsList = {
...
```
Comment (arkajyotiMukherjee) [inline src/modules/20-rbac/pages/UserDetails/__tests__/mock.ts:674]:
```suggestion
              'chaos_chaosgameday_edit',
```

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]

**Review by imrajdas — DISMISSED** [review]

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by abhinavrastogi-harness — DISMISSED** [review]

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by imrajdas — APPROVED** [review]

**Pull request merged**